src/main/java/mekanism/common/integration/multipart/MultipartTileNetworkJoiner.java

```diff
@@ -0,0 +1,83 @@
+package mekanism.common.integration.multipart;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import io.netty.buffer.ByteBuf;
+import mcmultipart.api.container.IMultipartContainer;
+import mcmultipart.api.slot.EnumFaceSlot;
+import mcmultipart.api.slot.IPartSlot;
+import mcmultipart.api.world.IMultipartBlockAccess;
+import mekanism.common.base.ITileNetwork;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.world.IBlockAccess;
+
+public class MultipartTileNetworkJoiner implements ITileNetwork
```

Comment:
`getNetworkedData` certainly, since it simply uses an `ArrayList` and it would be easy to insert a byte as the first object. Unfortunately, for the case of `handlePacketData` since it's using a stream and the last override is called first (that, of course, being the ones in the individual transmitter implementations) trying to clear the header inside `TileEntitySidedPipe` wouldn't work.

What I think I'll do is, in the morning, I'll write up the code to check whether or not the TE is part of a container and call the static wrapper method in `TileEntitySidedPipe.getNetworkedData` / `TileEntityGlowPanel.getNetworkedData` and then test it having `MultipartTileNetworkJoiner` remove those headers before they're passed on to `handlePacketData`. That could work and clean up a lot of the code.